networking_ccloud/ml2/agent/common/switch.py

```diff
@@ -111,3 +112,15 @@ def apply_config_update(self, config):
 
     def _apply_config_update(self, config):
         raise NotImplementedError
+
+    @run_in_executor('write')
+    def persist_config(self):
+        start_time = time.time()
+        try:
+            self._persist_config()
+            LOG.debug("Switch config of %s saved in %.2f", self, time.time() - start_time)
+        except Exception as e:
+            LOG.error("Saving switch config of %s failed in %.2f: %s", self, time.time() - start_time, e)
```

Comment:
You sure you don't want `LOG.exception`? `KeyError`s area otherwise quite hard to debug ...
```
>>> try:
...   {}['asdf']
... except Exception as e:
...   print(str(e))
... 
'asdf'
```

Reply:
I only expect there to be GNMI errors and I think these don't need a backtrace logged. We also already log the backtrace for GNMI commands. So until the switch config saving get's any more complex I'd leave it this way.